### PR TITLE
GitHub templates

### PR DESCRIPTION
**Pull request by mburns**
- adds Issue and Pull Request templates
- Updates `CONTRIBUTING.md` to explain how new Players can participate
  - Simplifies `README.md`
- Remove `templates/` as redundant to the documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,19 +16,23 @@ This is an instance of the game [Nomic](https://en.wikipedia.org/wiki/Nomic) dri
 Gameplay consists of:
 
 1. [Follow the existing rules](/rule101.md). Questions can be asked via [Issues](https://github.com/mburns/nomic/issues).
-2. A person [proposes a rule-change](#submitting-a-rule-change) to an existing (or entirely new) rule via Pull Request.
+2. A Player [proposes a rule-change](https://github.com/racker/chef/blob/master/.github/CONTRIBUTING.md) to an existing (or entirely new) rule via Pull Request.
```
Comment (jirwin) [inline README.md]: This doesn't look like the right url

File: README.md [hunk 2]
```diff
@@ -16,19 +16,23 @@ This is an instance of the game [Nomic](https://en.wikipedia.org/wiki/Nomic) dri
...
 3. [Players](https://github.com/mburns/nomic/wiki/Player) [discuss](/rule111.md) and [vote](/rule105.md) on proposals, earning [points in the process](/SCOREBOARD.md).
 
 The game is won when the first user reaches [+200 points](/rule208.md) or is [stuck on a turn that is an impossible position](/rule213.md) and it cannot be resolved through discussion or jurisdiction within the game.
 
 ### Joining the game
 
-Create a fork of this repo!
+1. [Vote](https://github.com/racker/chef/blob/master/.github/CONTRIBUTING.md#voting)
```
Comment (jirwin) [inline README.md]: same here

**Comment by jirwin** [comment]
Looks good other than some copypasta

**Comment by mburns** [comment]
Updated URLs.

**Pull request merged**